cmd/airflow.go

```diff
@@ -107,6 +107,7 @@ astro dev init --airflow-version 2.2.3
 	errPytestArgs          = errors.New("you can only pass one pytest file or directory")
 	buildSecrets           = []string{}
 	errNoCompose           = errors.New("cannot use '--compose-file' without '--compose' flag")
+	defaultWaitTime        = 5 * time.Minute
```

Comment:
I think the default should still be a minute or at best 2 minutes, the reason behind not having 5 minutes timeout earlier (and only restricting that to M1 with runtime version less than 6.x) was to have a faster response cycle back to the customer if things go wrong. I don't see a reason for increasing it based on the PR description, so maybe I missing some context.

Reply:
That would be fine too. I would be fine changing it back to 1, but figured we could take this opportunity to simplify things. Having a different timeout based on the host architecture and the runtime version feels a little awkward I think. I also noticed that the runtime version we're checking for with this M1 logic is no longer under LTS support, so it seemed like we might be able to clean this up a bit. What do you think @neel-astro @jeremybeard?

Reply:
I agree on cleaning up the bits we added around M1, but we bumped the timings to 5 minutes for M1 as an exception until 6.x, when runtime added native arm support. So I might favor simplifying things but keeping the default timing to 1 minute for all.

Reply:
This is done.